src/Interfaces/IShapeCharacter.ts

```diff
@@ -0,0 +1,7 @@
+export interface IShapeCharacter {
+    value: string | string[];
+    font: string;
+    style: string;
+    weight: string;
+    fill: boolean;
+}
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [file should end with a newline](https://app.codacy.com/manual/ar3s/tsparticles/pullRequest?prid=5098130)